### DISCUSSION TITLE
Server Actions ocassionally Crash and report HTTP 405 NOT ALLOWED

### DISCUSSION THREAD

**Discussion by hnykda** [vercel/next.js#59837] [Help]
### Summary

I am starting to be quite desperate. I can't find why, but our server actions just "sometimes" crash (<1% cases) with non descriptive error such as HTTP 405. I am unable to create a reproducible example, unfortunately.

Server logs just show this:
![image](https://github.com/vercel/next.js/assets/2741256/d7f79081-0a7e-4369-8029-371b08e978b1)

`FUNCTION_INVOCATION_FAILED: This Serverless Function has crashed`.

And on the client I see that the same query works just fine, then it returns HTTP 405 (not allowed) - which is definitely not what I would expect.

![image](https://github.com/vercel/next.js/assets/2741256/9f48c0a7-1998-41e6-a171-68d6425cb836)

### Additional information

```js
Any ideas what I might be doing wrong? I am on Nextjs 14.
```

**Comment by nicosammito** [comment]
Without seeing the code behind the server action it's quite hard to give a proper answer. But it seems that the server action itself fails

**Reply by hnykda** [reply]
I am making a DB request there, e.g. creating a new object via supabase. I guess if there is no obvious thing I might be doing wrong, something might silently fail in here then... 🤷 

```
export async function createNewForecast(
  userId: UUID,
  questionText: string,
  initialData?: ExtendedForecastUpdate,
) {
  const forecastId = nanoid(FORECAST_SESSION_ID_LENGTH);
  if (questionText.length < 5) {
    throw new Error('Question text must be at least 5 characters long');
  }
  const { error } = await supabase.from('forecasts').insert({
    user_id: userId,
    question_text: questionText,
    results: {},
    id: forecastId,
    ...initialData,
  });
  if (error) {
    console.error('in createNewForecast: ', error);
    throw error.message;
  }
  return { id: forecastId };
}
```
